### DISCUSSION TITLE
Is it or It will be possible change or remove the API routes prefix "/api/*"?

### DISCUSSION THREAD

**Discussion by glopezep** [nuxt/nuxt#18005] [Questions]
.

**Comment by TheAlexLichter** [comment]
Right now, this is not possible as the `api` prefix is harcoded:
https://github.com/nuxt/framework/blob/c80870ad8c4a1c1b044d7a5ae27e41e997f8a75f/packages/nitro/src/server/middleware.ts#L54

**Comment by wobsoriano** [comment]
You can manually add it to `serverMiddleware` for now

https://github.com/nuxt/framework/discussions/2573
